api/services/email_mailgun.js

```diff
@@ -79,24 +82,36 @@ module.exports = {
       return callback(undefined, 'Email sent')
     }
 
-    var addresses = R.map(R.objOf('address'), R.keys(data.email))
+    var shapeEmailBody = recipient =>
+      recipient.content.slice(1).join('\n\n')
 
-    var emailArray = R.zipWith(R.merge)(addresses)(R.values(data.email))
+    var grabSubjects = recipient =>
+      recipient.content[0]
 
-    var sendEmail = (recipient, cb) => {
-      var address = recipient.address
-      var emailBody = recipient.content.slice(1).join('\n\n')
-      var subject = recipient.content[0]
-      mg.sendText('messenger@friendsch.org', address, subject, emailBody, error =>
-        error ? cb({error: error.toString(), recipient: recipient.address}, null) : cb(null, address)
+    var toKey = R.compose(R.map(R.objOf('to')), R.keys)
+    var textKey = R.compose(R.map(R.objOf('text')), R.values, R.map(shapeEmailBody))
+    var subjectKey = R.compose(R.map(R.objOf('subject')), R.values, R.map(grabSubjects))
+
+    var emailsData = R.converge(R.zipWith(R.merge), [R.converge(R.zipWith(R.merge), [toKey, textKey]), subjectKey])(data.email)
```

Comment:
This might be better written as

```js
var emailsData = R.map(R.compose
  ( R.zipObj(['to', 'text', 'subject'])
  , R.mapObjIndexed((v, k) => [ k, shapeEmailBody(v), grabSubjects(v) ])
  )
)(data.email)
```

(I've not tested this but think it should work. Note also, at the very least mapping `shapeEmailBody` and `grabSubjects` should be moved to a compose with the `objOf`s to reduce iterations)